splice_machine/src/main/java/com/splicemachine/derby/iapi/sql/execute/RunningOperation.java

```diff
@@ -29,14 +29,15 @@ public RunningOperation(SpliceOperation operation,
                             UUID uuid, String rdbIntTkn) {
         this.operation = operation;
         this.thread =thread;
-        this.submittedTime = submittedTime;
+        this.submittedTime = (Date)submittedTime.clone();
```

Comment:
I would rather remove this clone, it shouldn't be too costly but for OLTP queries it could add up since it's in the hot path.

Reply:
this is a spotbugs issue, complaining about assigning a value that could be modified later. i can ignore it though.

Reply:
reverted and added ignores for Spotbugs EI_EXPOSE_REP and EI_EXPOSE_REP2:
EI_EXPOSE_REP2: May expose internal representation by incorporating
reference to mutable object

We might want to replace java.util.Date with java.time.Instance, see
https://stackoverflow.com/questions/36639154/convert-java-util-date-to-what-java-time-type